public/views/index.html

```diff
@@ -254,7 +278,7 @@
         var cssLink = document.createElement("link");
         cssLink.rel = 'stylesheet';
 
-        if (darkQuery.matches) {
+        if (false && darkQuery.matches) {
```

Comment:
```suggestion
        if (darkQuery.matches) {
```

Reply:
duh, sorry was trying to force it and forgot I left it in, thank you 🙏